src/server/room.js

```diff
@@ -44,11 +45,27 @@ class Room {
 
   constructor(id /* : string */) {
     this.id = id;
+    this.playerList = [];
+    this.teamNone = { ...defaultTeam};
```

Comment:
`playerList` should be a map (`{}`) with the userId as a key so it's easier to look up
also what is the use-case of playerList? don't we have all the players within the teams?

Reply:
i will remove teamNone. the idea was just to have some way to hold team-less players